LoopKitUI/Views/QuantityScheduleEditor.swift

```diff
@@ -33,7 +33,6 @@ struct QuantityScheduleEditor<ActionAreaContent: View>: View {
     var guardrailWarning: (_ crossedThresholds: [SafetyClassification.Threshold]) -> ActionAreaContent
     var savingMechanism: SavingMechanism<DailyQuantitySchedule<Double>>
     var mode: PresentationMode
-    var buttonText: Text
```

Comment:
This is stuff that should have been removed in my delivery limits PR but got overlooked